Gemfile.lock

```diff
@@ -405,69 +408,70 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
```

Comment:
### [json](https://flori.github.io/json)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.6.3 → 2.7.1

[_[change-log](https://github.com/flori/json/blob/master/CHANGES.md), [source-code](https://github.com/flori/json), [gem-diff](https://my.diffend.io/gems/json/2.6.3/2.7.1)_]





<details>
<summary>Commits</summary>

A change of **98** commits. See the full changes on [the compare page](https://github.com/flori/json/compare/v2.6.3...v2.7.1).

These are the first 10 commits:
- (f68bbd6) [Perf. improvements to Hash#to_json in pure implementation ge…](https://github.com/flori/json/commit/f68bbd67181718e20bc367a62407967e72efcb46)
- (9bef2bd) [Remove unnecessary initialization of create_id in JSON.parse…](https://github.com/flori/json/commit/9bef2bd53715ee0193000fb3a3b50b8cec067d8f)
- (cd8bbe5) [Call `super` in `included` hook](https://github.com/flori/json/commit/cd8bbe56a3d0a3eca1419ae850088ce204f12ee5)
- (a81bcc0) [tests/ractor_test.rb: make assert_separately available](https://github.com/flori/json/commit/a81bcc03281e940e4e23d4a158ef6d7e27c6d968)
- (f4739c9) [Fix homepage url in gemspec](https://github.com/flori/json/commit/f4739c9e49daa3bcbe31c2129286279e398bda26)
- (9463784) [Merge pull request #508 from unasuke/gemspec-homepage](https://github.com/flori/json/commit/9463784ef31318f2047e6d0754656765ca04f465)
- (42e9175) [Fix dead link to Ragel in README](https://github.com/flori/json/commit/42e91752ad9ad85280432863042a57c7327b4486)
- (fe4860f) [Merge pull request #509 from okuramasafumi/patch-1](https://github.com/flori/json/commit/fe4860f7c8af0f02bfae880e81ce092264e0ec56)
- (0a4e3ac) [[DOC] Fix yet another dead link to Ragel](https://github.com/flori/json/commit/0a4e3ac77477b353c433a0e8daf33993ec8b38cd)
- (6447b45) [Merge pull request #510 from nobu/fix-ragel-link](https://github.com/flori/json/commit/6447b451a805a0f2487ef881cd99c9171a767ccf)


</details>